frontend/server/src/main/resources/log4j2.xml

```diff
@@ -58,6 +58,17 @@
 			</Policies>
 			<DefaultRolloverStrategy max="5"/>
 		</RollingFile>
+		<ScriptAppenderSelector name="telemetry_metrics">
+				<Script language="JavaScript"><![CDATA[
+				java.lang.System.getenv("SM_TELEMETRY_LOG") == null ? "null_appender" : "remote_appender";]]>
+		</Script>
+		<AppenderSet>
+				<Null name="null_appender"/>
+		<File name="remote_appender" fileName="${env:SM_TELEMETRY_LOG}">
+				<PatternLayout pattern="%m%n"/>
+		</File>
+		</AppenderSet>
+		</ScriptAppenderSelector>
```

Comment:
it should be removed from default TorchServe log4j config. It will confuse community members. Nobody understand telemetry metrics